invenio_communities/static/templates/invenio_communities/facets.html

```diff
@@ -0,0 +1,53 @@
+<div ng-init="order = {'volume': {'title': 'Volume', subfacetorder:{'issue':{'title':'Issue'}}}}">
+<!-- Facets -->
+  <div ng-if="vm.invenioSearchResults.aggregations[key].buckets.length" ng-repeat="(key, obj) in order">
+    <div class="facet-group">
+    <h3>{{ obj.title }}</h3>
+      <ul class="list-unstyled terms" ng-controller="FacetsShowMoreController">
+        <li ng-repeat="item in vm.invenioSearchResults.aggregations[key].buckets | limitTo: facetResults">
+          <div class="checkbox checkbox-info checkbox-circle">
+          <input id="checkbox-{{item.key}}" class="include-facet" type="checkbox" ng-checked="handler[key].indexOf(item.key) > -1" ng-click="handleClick(key, item.key)" />
+          <label for="checkbox-{{item.key}}">
+            <span class="facet-title-truncate">{{ item.key }}</span>
+            <span class="facet-label pull-right term-count">{{ item.doc_count }}</span>
+          </label>
+          </div>
+          
+          
+          
+<!-- SubFacets -->
+            <div ng-if="item[subkey].buckets.length" ng-repeat="(subkey, subobj) in obj.subfacetorder">
+            <div class="facet-group">
+            <h4>{{ subobj.title }}</h4>
+              <ul class="list-unstyled terms" ng-controller="FacetsShowMoreController">
+                <li ng-repeat="subitem in item[subkey].buckets | limitTo: facetResults">
+                  <div class="checkbox checkbox-info checkbox-circle">
+                  <input id="checkbox-{{subitem.key}}" class="include-facet" type="checkbox" ng-checked="handler[subkey].indexOf(subitem.key) > -1" ng-click="!((handler[key].indexOf(item.key) > -1) !== (handler[subkey].indexOf(subitem.key) > -1)) && handleClick(key, item.key); handleClick(subkey, subitem.key)" />
```

Comment:
you know, angular is here to make simple and beautiful JS code (I love JS frameworks...)

so, globally, what it does... We have facets and subfacets, named "Volume" (top collection) and "Issue" (sub-collection). If you click on an issue facet, you want to search for this issue in the given volume.

So, if we have:
- Volume 1, Issue a
- Volume 1, Issue b
- Volume 2, Issue a

and you click on the first one, you want to search for issue a in volume 1, and not all issue a.

in the case volume 1 is already checked, we don't uncheck it, so we don't call the function "handleClick". Otherwise we do.
that's all :p